canutils/lely-canopen/0001-NuttX-port.patch

```diff
@@ -0,0 +1,512 @@
+From 2232cb2e64241fd56293a6bcc51365dd220bdd5d Mon Sep 17 00:00:00 2001
```

Comment:
could we upstream the change?

Reply:
This'll be my next step - try to upstream the changes. But I leave it for later (next month)